frontend/app/src/components/home/CurrentChatMessages.svelte

```diff
@@ -287,15 +287,32 @@
 
     function shouldLoadPreviousMessages() {
         morePrevAvailable = client.morePreviousMessagesAvailable(chat.chatId);
-        return !loadingPrev && calculateFromTop() < MESSAGE_LOAD_THRESHOLD && morePrevAvailable;
+        const insideLoadThreshold = calculateFromTop() < MESSAGE_LOAD_THRESHOLD;
+        const result = !loadingPrev && insideLoadThreshold && morePrevAvailable;
+        if (result) {
+            console.debug(
+                "SCROLL: shouldLoadPreviousMessages [loadingPrev] [insideLoadThreshold] [morePrevAvailable]",
```

Comment:
Given that we already check that `result` is true, the args we are printing are pointless since we know their values already.

Reply:
This is true! but I'm wondering if it isn't still nice to show them so that you don't need to look at the code

Reply:
SGTM